docs/cases/indicators-of-compromise.asciidoc

```diff
@@ -130,4 +130,41 @@ When you attach an indicator to a case, the indicator is added as a new comment
 To remove an indicator attached to a case, click *More actions* (*...​*) -> *Delete attachment* in the case comment. 
 
 [role="screenshot"]
-image::images/remove-indicator.png[Removing an indicator from a case]
\ No newline at end of file
+image::images/remove-indicator.png[Removing an indicator from a case]
+
+[discrete]
+[[add-indicator-to-blocklists]]
+== Add indicators to blocklists
+
+Add indicators to <<blocklists, blocklist>> to prevent selected applications from running on your hosts. When creating a blocklist, you can add indicators that have hash values for the MD5, SHA-1, or SHA-256 fields and are of the `file` indicator type.
+
+.Requirements
+[sidebar]
+--
+* In addition to configuring specific entries on the **Blocklist** page, you must also ensure that the blocklist is enabled on the {elastic-defend} integration policy in the <<malware-protection, Malware protection settings>>. This setting is enabled by default.
+
+* You must have the `superuser` {ref}/built-in-roles.html[built-in user role] to access this feature.
+--
+
+To add an indicator to a blocklist:
+
+. From the Indicators table, click the *More actions menu* (*...​*) menu, then select *Add blocklist entry*. Alternatively, open an indicator's details, then select *Take action > Add blocklist entry*. The *Add blocklist* flyout opens.
+
+. Fill in these fields in the **Details** section:
+.. `Name`: Enter a name to identify the indictor in the blocklist.
+.. `Description`: Enter a description to provide more information on the blocklist entry (optional).
+
+. In the **Conditions** section, verify that the information about the indicator is correct.
```

Comment:
@nastasha-solomon there must be some logic happening here, but when I look at the preview there is a lot more content under this `Conditions` section.
The point `#d` of that content mentions this
![Screenshot 2023-02-28 at 4 23 30 PM](https://user-images.githubusercontent.com/17276605/221995342-ed764109-b66b-4dda-bfb8-f42c19839730.png)
but in our scenario, the hash should be already populated in the input. I guess users can go and modify it if they want, but they shouldn't have too...

Reply:
You might be looking at an old version of the doc build. Let me rebuild the preview - give the PR about 5 mins to complete.

Reply:
I see that the build is finished, but I'm not sure how to find the new link. The one listed above in the description of in the github-actions comment is identical to before

Reply:
I'm seeing the same on my end, even after clearing my browser session cache. I'm rebuilding the HTML preview one more time to see if that fixes this.